docs/learn/documentation/versioned/jobs/logging.md

```diff
@@ -116,12 +116,14 @@ And then updating your log4j.xml to include the appender:
 
 #### Stream Log4j Appender
 
-Samza provides a StreamAppender to publish the logs into a specific system. You can specify the system name using "task.log4j.system" and change name of log stream with param 'StreamName'. Also, we have the [MDC](http://logback.qos.ch/manual/mdc.html) keys "containerName", "jobName" and "jobId", which help identify the source of the log. In order to use this appender, simply add:
+Samza provides a StreamAppender to publish the logs into a specific system. You can specify the system name using "task.log4j.system" and change name of log stream with param 'StreamName'. You can also specify the number of partitions for the log stream with param 'PartitionCount'; otherwise, the number of partitions will equal the number of containers configured for the job. Also, we have the [MDC](http://logback.qos.ch/manual/mdc.html) keys "containerName", "jobName" and "jobId", which help identify the source of the log. In order to use this appender, simply add:
```

Comment:
What happens when the number of containers changes (fairly common)?

PS: Unrelated but lets reword the "Also, we have the" to "The MDC contains the keys ... which may help identify ...". Also remove "simply"

Reply:
Nothing changes. Similar to how StreamAppender topics get auto-created. If the logging system is Kafka, the # of partitions is the default partition count configured on Kafka. I'll add that additional description and reword the other parts as well.